analyses/ner/config.yml

```diff
@@ -0,0 +1,25 @@
+#general country information
```

Comment:
We don't have configs in the `analyses` folder, only in the `src`. So this can be removed imo. 
Aah I now see that is in the template, sorry! That has to be updated, will do that tomorrow.

Reply:
@hannahker pointed me to the template folder. I take note the template is outdated (Hannah: FYI).

Reply:
Thanks for flagging this. Have just updated the template and making a PR now!